### DISCUSSION TITLE
Images in {{ content }}

### DISCUSSION THREAD

**Discussion by HarryFischer** [jekyll/jekyll#9073] [Q&A]
Hi –
Having some issues with how images are getting pulled through using Siteleaf -> Jekyll -> GHPages

I have the image added here:
![image](https://user-images.githubusercontent.com/20658471/173128105-ec57c277-8864-4d28-a7ae-478c705863bc.png)

and then added here:
![image](https://user-images.githubusercontent.com/20658471/173128118-c3d6abf0-5d17-4fd0-a2ec-ac9b4586652a.png)

But on the page I don't get the baseurl appended so the link is broken:
![image](https://user-images.githubusercontent.com/20658471/173128130-8619e429-6a9a-47b9-b2ca-0d52f5efc6ab.png)

It is also getting wrapped in a <p> which I don't want.

I tried {{ content | markdownify }} but no luck

Any ideas?

**Comment by pranabdas** [comment]
You can try following options: 
1. Use Jekyll filters: `![alt-text]({{ "/path/my-img.jpeg" | relative_url }})`
2. If you want absolute url instead: `![alt-text]({{ "/path/my-img.jpeg" | absolute_url }})`. You can read more here: <https://jekyllrb.com/docs/liquid/filters/>
3. I think in the older versions of Jekyll we needed liquid `prepend` filter <https://shopify.github.io/liquid/filters/prepend/>. `![alt-text]({{ "/path/my-img.jpeg" | prepend: site.base_url }})`

**Reply by HarryFischer** [reply]
Thanks @pranabdas 
Maybe I’m missing something … but the images are pulled in within {{ content }} so not sure how I’d append those filters?

**Reply by pranabdas** [reply]
@HarryFischer You add those filters where you have markdown syntax for images. You have image path inside `()` i.e., `(/uploads/xxxxxxx.jpeg)`, now it becomes `({{ '/uploads/xxxxxxx.jpeg' | relative_url }})`.

**Reply by HarryFischer** [reply]
Thanks @pranabdas 
That works for me
